azext_iot/tests/test_constants.py

```diff
@@ -44,3 +44,9 @@ class FileNames:
     central_query_response_file = "central/json/query_response.json"
     central_destination_file = "central/json/destination.json"
     central_export_file = "central/json/export.json"
+
+
+class CertificatesMessage:
+    invalidBase64 = "The certificate content is not a valid base64 string value"
+    unmatchedSegment = ("The certificate does not contain matched BEGIN and END segments, please either have both '-----BEGIN "
```

Comment:
Why not move and leverage these error message variables in the application code rather than only in test?

Reply:
add it to azext_iot/common/shared.py - you should add these as constants like how we do in azext_iot/digitaltwins/common.py

Reply:
Also we should use snake_case for variable names of multiple words (capitalized for constants) conforming to pep8 best practices. CapitalCase usually good for class names.